### PR TITLE
Update boskos readme to include request for more cpu quota

### PR DESCRIPTION
**Pull request by srinivashegde86**

### CODE REVIEW

**Comment by chaodaiG** [comment]
Did you mean `Update prow readme` in title?

**Comment by chaodaiG** [comment]
Have we already encountered this? Manually requesting cpu quota for all Boskos projects is going to be "interesting"

**Comment by srinivashegde86** [comment]
> Have we already encountered this?

I have been manually doing this for all the projects.

File: ci/prow_setup.md [hunk 1]
```diff
@@ -37,6 +37,13 @@
 
 1. Run `make update-boskos-config` to update the Boskos config.
 
+1. Increase the compute CPU quota for the project to 200. Go to
+   `https://pantheon.corp.google.com/iam-admin/quotas?project=<project_name>&service=compute.googleapis.com&metric=CPUs`
+   and click `Edit Quota`. This needs to be done manually and should be
```
Comment (chaodaiG) [inline ci/prow_setup.md]: I clicked on `Edit Quota` in knative-boskos-01, and it asked `Select one or more quotas to edit.`

Comment (srinivashegde86) [reply ci/prow_setup.md]: you need to select the box next to it and then click on edit quota

Comment (chaodaiG) [reply ci/prow_setup.md]: there are 20 boxes, each one represents different regions, I think we need to at least request quota for `us-central1`, `us-west1`, and `us-east1`

Comment (srinivashegde86) [reply ci/prow_setup.md]: up[dated to increase for 5 regions as we might run perf tests in 5 regions :)

File: ci/prow_setup.md [hunk 2]
```diff
@@ -37,6 +37,14 @@
 
 1. Run `make update-boskos-config` to update the Boskos config.
 
+1. Increase the compute CPU quota for the project to 200. Go to
+   `https://pantheon.corp.google.com/iam-admin/quotas?project=<project_name>&service=compute.googleapis.com&metric=CPUs`
```
Comment (adrcunha) [inline ci/prow_setup.md]: Please use console.cloud.

File: ci/prow_setup.md [hunk 3]
```diff
@@ -37,6 +37,14 @@
...
+   and click `Edit Quota`. Select atleast five regions to increase the quota
```
Comment (adrcunha) [inline ci/prow_setup.md]:
```suggestion
   and click `Edit Quota`. Select at least five regions to increase the quota
```

File: ci/prow_setup.md [hunk 4]
```diff
@@ -37,6 +37,14 @@
...
+   (`us-central1, us-west1, us-east1, europe-west1, asia-east1`). This needs
+   to be done manually and should be automatically approved once the request
+   is submitted. The request asks for a phone number and a reason. You can
```
Comment (adrcunha) [inline ci/prow_setup.md]: Replace the last two sentences with the suggested reason only.

**Comment by chaodaiG** [comment]
/lgtm
Did you do this for all 70 Boskos projects already?

**Comment by srinivashegde86** [comment]
I have done this for 45 projects. Not for the ones added today. Will do them next :)

File: ci/prow_setup.md (hunk 4 above)
Comment (adrcunha) [reply ci/prow_setup.md]: You did not address this.

**Comment by srinivashegde86** [comment]
I have updated all 70 projects to have the updated quota in us-central1.

**Comment by chaodaiG** [comment]
> I have updated all 70 projects to have the updated quota in us-central1.

Nice!
Just an fyi, performance test will still run out of cpu if stockout happened, since they are going to be outside of us-central1

File: ci/prow_setup.md (hunk 4 above)
Comment (adrcunha) [reply ci/prow_setup.md]: Resolved in https://github.com/knative/test-infra/pull/1098

**Pull request merged**